### PR TITLE
Migrate to pytest-selenium

### PR DESCRIPTION
**Pull request by davehunt**
This will definitely need rebasing after other pulls land.

Pinging @tojonmz and @rbillings for review.

### CODE REVIEW

**Comment by davehunt** [comment]
Also pinging @camd for visibility/review.

**Comment by rbillings** [comment]
I'm getting errors running the test now, not sure if it's due to my python version or not [2.7.9]. https://pastebin.mozilla.org/8851714

**Comment by davehunt** [comment]
Your version of pytest is out of date. Try wiping your virtual environment, or deleting it and starting with a fresh environment.

**Comment by rbillings** [comment]
I did that, twice, & still get the error. Any other suggestions?

**Comment by davehunt** [comment]
@rbillings these tests require pytest 2.4.2 or later, but it's picking up pytest 2.3.5 on your system. It's indicating that it's using a version in /Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages which suggests it's your system site packages and not a virtual environment. Maybe try `sudo uninstall pytest` whilst not in a virtual environment to try removing this old copy.

File: README.md [hunk 1]
```diff
@@ -65,22 +65,22 @@ To run all of the desktop tests against the default environment:
 $ py.test --driver firefox
 ```
 
-To run against a different environment, pass in a value for --baseurl, like so:
+To run against a different environment, pass in a value for --base-url, like so:
 
 ```bash
-$ py.test --baseurl https://treeherder.mozilla.org --driver firefox
+$ py.test --base-url https://treeherder.mozilla.org --driver firefox
```
Comment (bobsilverberg) [inline README.md]: I believe this needs to be `--driver Firefox` (upper case `F`)

**Comment by bobsilverberg** [comment]
The changes LGTM and the tests pass locally for me. Just the one change needed to the `readme`. r+ wc

**Comment by rbillings** [comment]
Tests now pass for me, changes look good.

**Pull request merged**